sw-emulator/lib/periph/src/root_bus.rs

```diff
@@ -331,17 +335,25 @@ impl CaliptraRootBus {
     pub const ICCM_SIZE: usize = 256 * 1024;
     pub const DCCM_SIZE: usize = 256 * 1024;
 
-    pub fn new(clock: &Clock, mut args: CaliptraRootBusArgs<'_>) -> Self {
+    pub const I3C_ERROR_IRQ: u8 = 17;
```

Comment:
I think these IRQs are not necessary for Caliptra.

Reply:
Removed.